src/api/package.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "api",
```

```suggestion
  "name": "gcp-pathfinder-api",
```

just my thoughts on branding